File: HomeWork5/src/main/java/org/phonebook/Phonebook.java
package org.phonebook;

import java.util.ArrayList;

public class Phonebook {

    public ArrayList<Contact> pb = new ArrayList();
    public void Addressbook(){
        pb.add(new Contact("Иванов Иван Иванович", "1234567890", "Иваново"));
        pb.add(new Contact("Сидоров Сидор Сидорович", "0987654321", "Сидорово"));
    }

    public void showAddressBook(){
        for(Contact i : pb){
            System.out.println(i.getName() + " " + i.getPhone() + " " + i.getCity());
        }
    }
}

File: HomeWork7/Phonebook.py
from Contact import Contact


class Phonebook:

    def __init__(self):
        self.pb = []

    def addressbook(self):

        self.pb.append(Contact("Иванов Иван Иванович", "1234567890", "Иваново"))

    def show_address_book(self):
        for i in self.pb:
            print(i.Name + " " + i.Phone + " " + i.City)
